packages/worker/src/api/routes/global/tests/users.spec.ts

```diff
@@ -242,6 +242,26 @@ describe("/api/global/users", () => {
       expect(response.body.message).toBe(`Unavailable`)
       expect(events.user.created).toBeCalledTimes(0)
     })
+
+    it("should not be able to create a user with the same email and different casing", async () => {
+      const user = structures.users.user()
+      await api.users.saveUser(user)
+
+      user.email = user.email.toUpperCase()
+      await api.users.saveUser(user)
+
+      expect(events.user.created).toBeCalledTimes(1)
```

Comment:
Should this be `toBeCalledTimes(0)`? Given the user already exists, it seems like it's not matching to the existing one

Reply:
it generates a UUID so it should always be called once, but the next one shouldn't be called because it's been uppercased.